Misc/NEWS.d/next/Windows/2021-04-08-00-36-37.bpo-34816.4Xe0id.rst

```diff
@@ -0,0 +1,2 @@
+``hasattr(ctypes.windll, 'nonexistant')`` now returns `False` instead of raising exc:`OSError`.

```

Comment:
I still messed it up :sweat_smile: 

```suggestion
``hasattr(ctypes.windll, 'nonexistant')`` now returns `False` instead of raising `exc:`OSError`.

```